### ISSUE TITLE
plot_implicit error for Xor

### ISSUE THREAD

**Issue by sylee957** [sympy/sympy#17719]
This can be an issue that needs to be resolved after addressing #17710 
`xor` is not properly comprehended for `plot_implicit` now, ~~but it can be beyond the scope for what I want to do in #17711~~

```
from sympy import *

x, y = symbols('x y')

a = (x - 1)**2 + y**2 < 2
b = (x + 1)**2 + y**2 < 2

plot_implicit(a ^ b, x, y)
```

**Comment by Tamoghna-Bhattacharjee** [comment]
make me confirm that the plot you want here is 
union(a,b) - intersection(a,b)

**Comment by sylee957** [comment]
@Tamoghna-Bhattacharjee Yes, it is what I would expect.

**Comment by Tamoghna-Bhattacharjee** [comment]
can anyone guide me where should i start from.

**Comment by sylee957** [comment]
It is not a difficult issue to fix, from my base #17711 

```diff
$ git diff
diff --git a/sympy/plotting/intervalmath/interval_membership.py b/sympy/plotting/intervalmath/interval_membership.py
index a7aca9e4a0..6908b2b73b 100644
--- a/sympy/plotting/intervalmath/interval_membership.py
+++ b/sympy/plotting/intervalmath/interval_membership.py
@@ -69,7 +69,7 @@ def __xor__(self, other):

         a1, b1 = self
         a2, b2 = other
-        return intervalMembership(fuzzy_xor([a1, a2]), fuzzy_xor([b1, b2]))
+        return intervalMembership(fuzzy_xor([a1, a2]), fuzzy_and([b1, b2]))

     def __eq__(self, other):
         return self._wrapped == other
diff --git a/sympy/printing/precedence.py b/sympy/printing/precedence.py
index 078621f519..21e3f9c7e5 100644
--- a/sympy/printing/precedence.py
--- a/sympy/printing/precedence.py
+++ b/sympy/printing/precedence.py
@@ -18,6 +18,7 @@
     "Not": 100,
     "Atom": 1000,
     "BitwiseOr": 36,
+    "BitwiseXor": 37,
     "BitwiseAnd": 38
 }

diff --git a/sympy/printing/str.py b/sympy/printing/str.py
index 2ec879b0aa..a3f59ecb51 100644
--- a/sympy/printing/str.py
+++ b/sympy/printing/str.py
@@ -82,6 +82,9 @@ def _print_And(self, expr):
     def _print_Or(self, expr):
         return self.stringify(expr.args, " | ", PRECEDENCE["BitwiseOr"])

+    def _print_Xor(self, expr):
+        return self.stringify(expr.args, " ^ ", PRECEDENCE["BitwiseXor"])
+
     def _print_AppliedPredicate(self, expr):
         return '%s(%s)' % (self._print(expr.func), self._print(expr.arg))
:
```

I think that it's a self-contained issue of `StrPrinter` for not having a proper method of printing `Xor` to python strings, but I'm considering to merge this to my patch in #17711 for a better showcase.

Thanks for your time.
But unfortunately, you may have to find an other issue to work with, unless you have other suggestions.
